prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -5,21 +5,20 @@ postsubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/gcp-guest/osconfig-agent-build:latest
-        imagePullPolicy: Always
+      - image: gcr.io/compute-image-tools-test/osconfig-agent-build:latest
```

Comment:
build your image into gcp-guest registry, please

Reply:
i had this idea initially, but then since we were using `compute-image-tools-test` for other jobs, i decided to continue. Are we planning to move everything to gcp-guest? I would actually like that. doesn't make sense to have cross-project dependency